C/tasks/update_alerts/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+cmake_minimum_required (VERSION 2.8.8)
+project (update_alerts)
```

Comment:
I think I would prefer a nam like "checkForUpdates" as "update alerts" sounds like is is updating an alert, not rating an alert if updates are available.